### ISSUE TITLE
java.lang.NullPointerException: null when refreshing items

### ISSUE THREAD

**Issue by fmatray** [openhab/openhab#3218]
Hello,
I use openhab 1.7.1 on a raspberry with debian jessie using the debian repository.
Sometimes, after a modification of an item, I get an java exception. 

```
2015-09-27 00:00:46.299 [INFO ] [c.internal.ModelRepositoryImpl] - Refreshing model 'maisondulion.items'
2015-09-27 00:00:46.511 [ERROR] [.o.m.c.i.folder.FolderObserver] - An unexpected exception has occured
java.lang.NullPointerException: null
        at org.openhab.model.item.binding.AbstractGenericBindingProvider.removeConfigurations(AbstractGenericBindingProvider.java:97) ~[na:na]
        at org.openhab.model.item.internal.GenericItemProvider.processBindingConfigsFromModel(GenericItemProvider.java:164) ~[na:na]
        at org.openhab.model.item.internal.GenericItemProvider.modelChanged(GenericItemProvider.java:390) ~[na:na]
        at org.openhab.model.core.internal.ModelRepositoryImpl.notifyListeners(ModelRepositoryImpl.java:159) ~[na:na]
        at org.openhab.model.core.internal.ModelRepositoryImpl.addOrRefreshModel(ModelRepositoryImpl.java:100) ~[na:na]
        at org.openhab.model.core.internal.folder.FolderObserver.checkFolder(FolderObserver.java:142) ~[na:na]
        at org.openhab.model.core.internal.folder.FolderObserver.run(FolderObserver.java:99) ~[na:na]
201

**Comment by watou** [comment]
I think this is the same issue as #2842 and #1419.  I submitted PR #3016 in an attempt to fix it.

**Comment by fmatray** [comment]
Thanks. 
So I'm waiting for next version :)

**Comment by watou** [comment]
Let's leave this issue open until we know it's fixed!  You can try a Cloudbees build from recent hours to see if the problem is gone.  Thanks!

**Comment by steve-bate** [comment]
@fmatray Have you had a chance to try the fix for #3218?

**Comment by steve-bate** [comment]
Closing this issue because of lack of activity.
